src/adapter/src/catalog.rs

```diff
@@ -2952,6 +2952,7 @@ impl<S: Append> Catalog<S> {
             CatalogType::Uuid => CatalogType::Uuid,
             CatalogType::VarChar => CatalogType::VarChar,
             CatalogType::Int2Vector => CatalogType::Int2Vector,
+            CatalogType::Int4Range => CatalogType::Int4Range,
```

Comment:
Are ranges composed of 2 int4 numbers? Kafka offsets are 64-bit

Reply:
Ranges are for the Kafka partition IDs; the offsets are stored separately as 64-bit numbers separately